sql/sql_expression.cc

```diff
@@ -2960,6 +2960,20 @@ printJsonTyped(const SelectExpression * val,
     else context.writeStringUtf8(val->surface);
 }
 
+/*****************************************************************************/
+/* JOIN QUALIFICATION                                                        */
+/*****************************************************************************/
+
+DEFINE_ENUM_DESCRIPTION(JoinQualification);
+
+JoinQualificationDescription::
+JoinQualificationDescription()
+{
+    addValue("JOIN_INNER", JOIN_INNER, "Inner join");
```

Comment:
This is an internal class, which is only printed out for debugging.  It's not part of the MLDB API so it probably doesn't really matter.